include/wx/log.h

```diff
@@ -406,7 +406,7 @@ class WXDLLIMPEXP_BASE wxLog
     // is logging of messages from this component enabled at this level?
     //
     // usually always called with wxLOG_COMPONENT as second argument
-    static bool IsLevelEnabled(wxLogLevel level, wxString component)
+    static bool IsLevelEnabled(wxLogLevel level, const wxString& component)
```

Comment:
Have you seen the comment just above? I'm pretty sure that this improvement turns out to be a minor pessimization with C++11 compilers.

Reply:
The comment just above talks about function GetComponentLevel, i that case it make sense. But here we have double object coping. In case of IsEnabled() return false it doesn't make any sense at all.

Reply:
What do you thing about cppcheck?

Reply:
The same comment obviously applies to this function, which just calls `GetComponentLevel()`.

I have no idea about what do you mean by your questions (some later edited to say something completely different) but please don't submit PRs with optimizations without profiling the code before and after your changes. TIA!